src/Configurations/test/Responses.test.js

```diff
@@ -24,31 +37,41 @@ jest.mock('@folio/stripes/core', () => ({
 }));
 
 
-describe('EditorLayer', () => {
-  it('`useSingleQuery` fires showCalloutHook if any error', async () => {
-    server.use(rest.get(url.configurations.single, ERROR_RESPONSE));
+const renderDetailsPane = async () => {
+  renderConfigurations();
 
-    const mockShowCallout = jest.fn().mockImplementation(() => jest.fn());
+  const cell = await screen.findByRole('gridcell', { name: 'RS1' });
+  user.click(cell);
+};
 
-    jest.spyOn(components, 'useShowCallout').mockImplementation(() => mockShowCallout);
 
-    renderConfigurations('/1/edit');
+describe('Fetching single configuration', async () => {
+  const mockShowCallout = jest.fn();
 
-    await waitFor(() => expect(mockShowCallout).toBeCalledWith({ 'messageId': 'ui-remote-storage.error', 'type': 'error' }));
+  beforeEach(async () => {
+    jest.spyOn(components, 'useShowCallout').mockImplementation(() => mockShowCallout);
   });
```

Comment:
This way we use the same `mockShowCallout` over number of tests.
So it needs to be [reset](https://jestjs.io/docs/mock-function-api#mockfnmockreset) before each test IMO.